srcalc/src/main/webapp/WEB-INF/views/displayResults.jsp

```diff
@@ -27,13 +27,14 @@ Note: this view is a fusion of two different objects:
     </tr>
     </c:forEach>
     </table>
-    <div class="actionButtons">
     <h3>Results</h3>
-    <ol>
+    <!-- The outcomes are actually sorted by model name, but use a <ul> anyway. -->
+    <ul id="outcomesList">
     <c:forEach var="outcome" items="${result.outcomes}">
-    <li>${outcome.key}: <fmt:formatNumber value="${outcome.value * 100}" minFractionDigits="1" maxFractionDigits="1" />%</li>
+    <li>${outcome.key}: <fmt:formatNumber value="${outcome.value}" type="percent" minFractionDigits="1" maxFractionDigits="1" /></li>
```

Comment:
Me neither until a few weeks ago, which is why I wrote it in a more awkward way before.